packages/widget/src/components/transfer/fungible/fungible-token-transfer.ts

```diff
@@ -85,6 +88,14 @@ export class FungibleTokenTransfer extends BaseComponent {
     )!;
   }
 
+  protected updated(changedProperties: PropertyValues): void {
+    if (changedProperties.has('network')) {
+      void this.walletController.switchChain(this.network?.chainId as number);
+      // reset the selector upon changing the network
+      this.transferController.onDestinationNetworkSelected(undefined);
```

Comment:
this should be inside transferController. It would also be better UX if you reset selector only if currently selected destination is not supported route